client/package.json

```diff
@@ -9,6 +9,7 @@
     "@material-ui/lab": "^4.0.0-alpha.33",
     "@syncfusion/ej2-react-grids": "^17.3.34",
     "autosuggest-highlight": "^3.1.1",
+    "axios": "^0.19.2",
```

Comment:
I noticed this was missing in the `client/package.json` so I added it in. i noticed there's a number of client dependencies in the server level `package.json` as well, which you prob want to clean up at some point.